test/createCycleMiddleware.test.js

```diff
@@ -54,9 +60,9 @@ describe('Redux cycle middleware', () => {
       const pong$ = sources.ACTION
         .filter(action => action.type === 'PING')
         .map(() =>
-            xs.periodic(10000)
-                .take(1)
-                .mapTo({ type: 'PONG' })
+          xs.periodic(10000)
```

Comment:
You could also potentially use `@cycle/time` for these tests if you'd like them to best fast 😄